includes/header.html

```diff
@@ -1,21 +1,24 @@
-    <header class="header">
-      <div class="container">
-        <a class="header-logo no-underline-hover" href="index.html">
-          <img src="images/header-logo.png" class="header-logo-image"/>
-        </a>
-
-        <nav class="header-nav">
-          <a class="header-footer-nav-link" href="https://github.com/eiffel-community/eiffel">Protocol</a>
-          <a class="header-footer-nav-link" href="https://eiffel-community.github.io">Eiffel Community</a>
-          <a class="header-footer-nav-link no-underline-hover" href="https://www.youtube.com/channel/UC3Zx84ecQ1jyGSc9709xrZg">
-            <img src="images/youtube-channel-icon.png" class="header-nav-link-image"/>
-          </a>
-          <a class="header-footer-nav-link no-underline-hover" href="https://github.com/eiffel-community/eiffel-sepia">
-            <img src="images/github-organization-icon.png" class="header-nav-link-image"/>
+    <header>
+      <nav class="nav">
+        <div class="nav-left">
+          <a class="no-left-padding" href="index.html">
+            <img src="images/header-logo.png"/>
           </a>
-          <a class="header-footer-nav-link no-underline-hover" href="https://groups.google.com/forum/#!forum/eiffel-community">
-            <img src="images/google-group-icon.png" class="header-nav-link-image"/>
-          </a>
-        </nav>
-      </div>
-    </header>
+        </div>
+
+        <div class="nav-right">
+            <a href="https://github.com/eiffel-community/eiffel">Protocol</a>
+            <a href="https://eiffel-community.github.io">Eiffel Community</a>
+            <a class="no-shrink"
+               href="https://www.youtube.com/channel/UC3Zx84ecQ1jyGSc9709xrZg">
+              <img src="images/youtube-channel-icon.png" class="header-nav-link-image"/>
+            </a>
+            <a class="no-shrink" href="https://github.com/eiffel-community/eiffel-sepia">
+              <img src="images/github-organization-icon.png" class="header-nav-link-image"/>
+            </a>
+            <a class="no-shrink" href="https://groups.google.com/forum/#!forum/eiffel-community">
+              <img src="images/google-group-icon.png" class="header-nav-link-image"/>
+            </a>
+        </div>
+      </nav>
+    </header>
```

Comment:
Nit: File should end with a newline character.

Reply:
Fixed in 71136f8